client/src/components/navbar/Navbar.js

```diff
@@ -47,14 +48,18 @@ const useStyles = makeStyles((theme) => ({
   },
   authLinkStyling: {
     textDecoration: "none",
-    marginRight: "30px",
+    marginRight: "20px",
     marginLeft: "20px",
   },
   signupLinkStyling: {
     textDecorationColor: "black",
   },
   removeTextDecoration: {
     textDecoration: "none",
+    marginLeft: "15px",
+  },
+  icons: {
+    color: "black",
```

Comment:
Nice job with the  icons but you probably want to use _theme.spacing_ for margins because on small screens the navbar is off a lil bit
![image](https://user-images.githubusercontent.com/24212950/107409359-143eb200-6ada-11eb-9542-544d2176a879.png)

Reply:
theme.spacing is the same margin in fact, just every point of spacing gives you 8px, e.g. theme.spacing(2) = 16px, theme.spacing(3) = 24px, etc.
The problem here is we are trying to place 6 icons in navbar and that's a lot. My concern is if we assign margin of a smaller value then icons would be placed too close to each other making it hard to point to a correct icon
P.S. we need to fix mobile listings, they are off to the left